packages/components/src/utils/index.tsx

```diff
@@ -66,3 +75,25 @@ export function PressableOpacityStyle(
 
   return ({ pressed }) => [{ opacity: pressed ? 0.2 : 1 }]
 }
+
+export type SpacerProps = {
+  /** Size of spacer, default 10 */
+  size?: number
+  /** True for horizontal spacing */
+  horizontal?: boolean
+}
+
+/** Convenience component for handling spacing without managing margin/padding between elements */
+export const Spacer: FC<SpacerProps> = ({
+  size = 10,
+  horizontal = false
+}) => {
+  return (
+    <View
+      style={{
+        width: horizontal ? size : 'auto',
+        height: !horizontal ? size : 'auto',
+      }}
+    />
+  )
+}
```

Comment:
Definitely see how this will be useful moving forward. What do you think about it living in its own file in `components/common` or `components/utils`? Feel like it's a little different than the utils we have so far. 

The organization of this utils folder is probably something we should think about as we add more so `index.tsx`.

Reply:
I was thinking of it like Icon--its' own component (if very simple one), but not a fully documented one and one I thought made sense to keep hidden away in the utils until the appropriate sizing tokens were established. Although it works just fine without the tokens so it could be made available now.

May be part of the larger question around how we want to expose utils generally--like the useIsScreenReaderEnabled hook I left despite it no longer being used with inline Link going away (for now) because it'd doubtlessly be helpful for apps besides flagship (and technically allow flagship some cleanup too).

Maybe split off a separate ticket to move it either to be the Spacer component or decide if we want a `components/common` potpourri of little things.

Reply:
Created ticket #257 for this. 

After thinking on it, wrote the ticket to call for a `Spacer/Spacer.tsx` structure as I didn't like the idea of having common/util "components" since Alert is directly using Button and it feels like we could get into a space of splitting hairs around use cases and making judgment calls for a delineation we already have with Icon of "proper" components have doc site documentation while more technical helper/convenience components only appear in Storybook to highlight their existence to technical folks. I think we'll benefit from the the simplicity long term of handling even simple components like any other component (aside from not documenting them fully).